tests/restart/25-hold-suite.t

```diff
@@ -31,7 +31,7 @@ else
     cmp_ok 'suite-is-held.out' <<<'1'
 fi
 T1_2016_PID="$(sed -n 's/CYLC_JOB_PID=//p' "${SUITE_RUN_DIR}/log/job/2016/t1/01/job.status")"
-poll ! ps "${T1_2016_PID}" 2>'/dev/null'
+poll ps "${T1_2016_PID}"
 cylc restart "${SUITE_NAME}" --debug --no-detach 1>"${TEST_NAME_BASE}-restart.out" 2>&1 &
```

Comment:
The `poll` function polls for the command to return a non-zero value. This line is supposed to wait on a process to finish. The incorrect `!` here causes test slowness (always times out) and since #3288 (currently only on 7.8.x) test failure on that timeout, and `2>/dev/null` removed the new timeout message.

Reply:
(So this test was broken on 7.8.x by #3288)